BaikalStandalone/Application/cl_render.cpp

```diff
@@ -52,40 +49,43 @@ namespace Baikal
     namespace
     {
         constexpr float kGamma = 2.2f;
-#ifdef ENABLE_DENOISER
-        std::unique_ptr<RendererOutputAccessor> m_output_accessor;
-        const std::size_t m_dump_period = 20;
+#ifdef IMAGE_DUMP_PATH
+        std::unique_ptr<RendererOutputAccessor> s_output_accessor;
 #endif
     }
 
     AppClRender::AppClRender(AppSettings& settings, GLuint tex)
-    : m_tex(tex), m_output_type(Renderer::OutputType::kColor), m_frame_count(0)
+    : m_tex(tex)
     {
         InitCl(settings, m_tex);
 
-#ifdef ENABLE_DENOISER
-        switch (settings.denoiser_type)
+        m_denoiser_type = settings.denoiser_type;
```

Comment:
May be after "switch" ? If exception will be raised and client will catch exception then m_denoiser_type will be wrong